db/migrate/20221207084440_add_user_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] The like table should have an `author_id`, not a `user_id`. That means this reference should be `author`, not `user`. Kindly change that. You can point rails to the user table using the `to_table` option for the foreign key attribute.

  See this answer: [https://stackoverflow.com/a/42056089](https://stackoverflow.com/a/42056089)